shim/src/main.rs

```diff
@@ -211,8 +213,20 @@ fn run_correct_turbo(repo_root: &Path, args: Vec<String>) -> Result<i32> {
     Ok(command.wait()?.code().unwrap_or(2))
 }
 
+fn get_version() -> &'static str {
+    include_str!("../../version.txt")
+        .split_once('\n')
+        .expect("Failed to read version from version.txt")
+        .0
+}
+
 fn main() -> Result<()> {
     let clap_args = Args::parse();
+    // Quick fix because --version doesn't work with ignore_errors in clap.
```

Comment:
Quick fix implies to me that there's some follow up work here. If that's not the case, I'd just say 

```suggestion
    // --version flag doesn't work with ignore_errors in clap, so we have to handle it manually
```

Reply:
Well there is some follow up work when clap fixes this bug, but sure, makes sense